openqa-prepare-mm-setup

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env perl
+# Copyright 2020 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+use strict;
+
+system("zypper in -y os-autoinst-openvswitch");
+system("systemctl enable --now os-autoinst-openvswitch");
+
+my $bridge = 1;
+my $openvswitch = "/etc/sysconfig/os-autoinst-openvswitch";
+open(file, '>', $openvswitch) or die "Failed to open $openvswitch for writing!";
```

Comment:
Lexical filehandles should be used instead of bareword filehandles:

```suggestion
open my $fh, '>', $openvswitch or die "Failed to open $openvswitch for writing!";
```